Dockerfile

```diff
@@ -1,7 +1,31 @@
-FROM elixir:1.13.0-alpine AS build
```

Comment:
From `mix phx.gen.release --docker` with modifications on content building, fly.io networking, and static file hosting.